sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -391,6 +391,29 @@ class HiveDDLSuite
     }
   }
 
+  test("create view with mismatched schema") {
```

Comment:
Without enabling Hive supports, we are unable to `CREATE VIEW` and then `SELECT VIEW`.